chpl/chpl-service/src/main/java/gov/healthit/chpl/activity/ProductActivityMetadataBuilder.java

```diff
@@ -13,26 +13,25 @@
 import gov.healthit.chpl.dao.DeveloperDAO;
 import gov.healthit.chpl.domain.activity.ActivityCategory;
 import gov.healthit.chpl.domain.activity.ActivityMetadata;
-import gov.healthit.chpl.domain.activity.DeveloperActivityMetadata;
 import gov.healthit.chpl.domain.activity.ProductActivityMetadata;
 import gov.healthit.chpl.dto.ActivityDTO;
 import gov.healthit.chpl.dto.DeveloperDTO;
 import gov.healthit.chpl.dto.ProductDTO;
 
 @Component("productActivityMetadataBuilder")
 public class ProductActivityMetadataBuilder extends ActivityMetadataBuilder {
-    private static final Logger LOGGER = LogManager.getLogger(ProductActivityMetadataBuilder.class);
+    private static Logger LOGGER = LogManager.getLogger(ProductActivityMetadataBuilder.class);
```

Comment:
Do we want to use the `@Log4j2` annotation from the Lombok library?  There are few other places in this PR with the same comment.